### PR TITLE
Add IsCableLift

### PR DESCRIPTION
**Pull request by frutiemax**

### CODE REVIEW

**Comment by spacek531** [comment]
Should we make a cable lift object...?

**Comment by frutiemax** [comment]
> Should we make a cable lift object...?

Maybe since it doesn't use an entry... I'm not sure how to tackle this, the vehicle code is so broken, why is it checking for every state when it's actually impossible to get in that state in the first place...

File: src/openrct2/ride/Vehicle.cpp [hunk 1]
```diff
@@ -1854,7 +1854,7 @@ void Vehicle::GetLiftHillSound(Ride* curRide, SoundIdVolume& curSound)
 void Vehicle::Update()
 {
     // The cable lift uses a ride entry index of NULL
```
Comment (Gymnasiast) [inline src/openrct2/ride/Vehicle.cpp:1856]: This comment is no longer needed.

Comment (Gymnasiast) [reply src/openrct2/ride/Vehicle.cpp:1856]:
```suggestion
```

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**